test/functional/importmulti.py

```diff
@@ -12,8 +12,8 @@ def __init__(self):
         self.num_nodes = 2
         self.setup_clean_chain = True
 
-    def setup_network(self, split=False):
-        self.nodes = start_nodes(2, self.options.tmpdir)
+    def setup_network(self):
+        self.setup_nodes()
```

Comment:
`setup_network()` in the base class also connects the nodes, which we don't want to do for this test.